RFC initial commitments - draft Oct 02 2024

Cofson opened the pull request.

Cofson committed:
RFC initial commitments - draft Oct 02 2024

kaiserd reviewed (commented):
Thank you.
Comments inline.
The waku commitment is missing.
Generally, the commitments have to be more clear and measurable.

Cofson committed:
RFC-commitments-patch1

Cofson committed:
rfc-commitments-patch2

- Edited according to markdown
- Fixed some of the waku descriptions
- Added a new waku commitment (split from previous one)

Cofson committed:
Update rfc-culture-establishment-2024q4.md

Small change

kaiserd reviewed (approved):
Thank you :).
comments will be addressed in a follow up.
